packages/react-components/react-components/src/index.ts

```diff
@@ -1423,3 +1423,67 @@ export {
   useInfoLabel_unstable,
 } from '@fluentui/react-infolabel';
 export type { InfoLabelProps, InfoLabelSlots, InfoLabelState } from '@fluentui/react-infolabel';
+
+export {
```

Comment:
we need to mark all `unstable` Drawer exports as deprecated ( `@deprecated` import drawer from `@fluentui/react-components`